BSDmakefile

```diff
@@ -42,13 +42,13 @@ GARGS = "--no-print-directory"
 
 # The GNU convention is to use the lowercased `prefix` variable/macro to
 # specify the installation directory. Humor them.
-GPREFIX = ""
+GPREFIX =
```

Comment:
Should we maybe use `:=` syntax, given that this is a makefile?

Reply:
I guess there is no actual difference for empty variable, so let's keep it I guess.

https://stackoverflow.com/a/448939/808699